device/core/src/device_client.ts

```diff
@@ -96,21 +96,41 @@ export class Client extends InternalClient {
    *
    * *Note: After calling this method the Client object cannot be reused.*
    *
-   * @param closeCallback Function to call once the transport is disconnected and the client closed.
+   * @param {Callback<results.Disconnected>} closeCallback Function to call once the transport is disconnected and the client closed.
```

Comment:
Here's sample documentation. I didn't add it to rest of the method as I want to hear feedback first.